clients/privacy-center/middleware.ts

```diff
@@ -0,0 +1,17 @@
+import { NextResponse } from "next/server";
+
+export default function middleware(request) {
+  const start = Date.now();
+  const response = NextResponse.next();
+  const stop = Date.now();
+  const handler_time = stop - start;
+
+  const log_dict = {
+    method: request.method,
+    status_code: response.status,
+    handler_time: `${handler_time}ms`,
+    path: request.destination,
```

Comment:
I think this is the right attribute? It always is a blank string since there is only one endpoint I guess?

Reply:
I think you want to use `request.nextUrl.pathname` instead of `request.destination`. `request.nextUrl.pathname` will return the route that was called (`/api/fides-js`). 

`request.destination` is a little different. Here are the docs for it: `Returns the kind of resource requested by the request, e.g., "document" or "script".`


https://nextjs.org/docs/app/api-reference/functions/next-request#nexturl
https://developer.mozilla.org/en-US/docs/Web/API/Request/destination